public/elements/seed-footer/seed-footer.scss

```diff
@@ -28,19 +28,33 @@ $predix-font-family : "GE Inspira Sans", sans-serif !default;
 $px-page-link-color: $gray5;
 
 .footer__container {
-  height: 4em;
+  background: var(--px-seed-footer-background-color);
+  height: 60px;
+  padding-left: 15px;
```

Comment:
All spacing should be derived from https://www.predix-ui.com/?show=px-spacing-design&type=css. Fine for this release though. 

I added an issue: https://github.com/PredixDev/predix-seed/issues/84